src/end.sol

```diff
@@ -148,6 +148,11 @@ interface SpotLike {
     all auctions are in the reverse (`dent`) phase. There are two ways
     of ensuring this:
 
+    NOTE: In the event there's a system surplus, and there are no
+    under-collateralised vaults remaining, one must skim() large CDPs in order
```

Comment:
Is it only _large_ CDPs?  Seems like if we need 0 surplus we need to `skim` all non-empty CDPs.

Reply:
We shouldn't need to `skim()` everything, as we just need enough to collect fees get surplus to 0.  The surplus cap means some of those fees are owed, but were sent off to surplus auctions.  That is, `surplus < all fees owed`.